registry/tracecat_registry/integrations/velociraptor/velociraptor.py

```diff
@@ -9,23 +9,28 @@
 import grpc
 import yaml
 from pydantic import Field
-from pyvelociraptor import api_pb2, api_pb2_grpc
 
 from tracecat_registry import RegistrySecret, registry, secrets
+from tracecat_registry.integrations.velociraptor import api_pb2, api_pb2_grpc
 
 velociraptor_secret = RegistrySecret(
     name="velociraptor_ssl",
     keys=["CONFIGURATION"],
+    optional_keys=["ORGANIZATION_ID"],
```

Comment:
for clarity, can these be namespaced? `VELOCIRAPTOR_ *`

and since `CONFIGURATION` is b64 encoded, can we do something like `B64_CONFIGURATION`? We don't currently have docs for each secret so ideally they're self documenting

Reply:
feel free to ignore